### PR TITLE
Fix random spelling errors

### PR DESCRIPTION
**Pull request by joaofnfernandes**

### CODE REVIEW

File: kitematic/known-issues.md [hunk 1]
```diff
@@ -38,4 +38,4 @@ if you're looking to help fix specific issues around VM provisioning.
 ## View All Issues
 
 For a full list of Kitematic bugs or issues see our <a href="https://github.com/kitematic/kitematic/issues?q=is%3Aopen+is%3Aissue+label%3Abug">existing GitHub
-issues</a> labelled as `bug`.
+issues</a> labeled as `bug`.
```
Comment (mstanleyjones) [inline kitematic/known-issues.md]: This one could go either way! Do we address it in the style guide?

File: datacenter/dtr/2.2/guides/admin/configure/garbage-collection.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ UTC time (unless the system time is mounted), therefore remember that the cron
 schedule will run based off of UTC time when configuring.
 
 GC puts DTR into read only mode; pulls succeed while pushes fail.  Pushing an
-image while GC runs may lead to undefined behaviour and data loss, therefore
+image while GC runs may lead to undefined behavior and data loss, therefore
```
Comment (mstanleyjones) [inline datacenter/dtr/2.2/guides/admin/configure/garbage-collection.md]: US vs CW English. Do we specify US in the style guide? Same comment for "labelled", "cancelled" below.

**Comment by joaofnfernandes** [comment]
Ye, I ran the linter in US mode instead of UK. Given that Docker is an international project/company I think we should strive to conform to that.

I didn't find many instances of this, so I think we can abstain for now from updating the style guide, since most people are already conforming.

**Comment by johndmulhausen** [comment]
This is great, you'll have to let the team know what your linter process was, @joaofnfernandes :)

**Pull request merged**